app/assets/javascripts/application.js.erb

```diff
@@ -488,6 +502,10 @@ onPageLoad(() => {
     parentSelector(event.target, '.markdown_help_toggler').querySelector('.markdown_help').classList.toggle('display-block');
   });
 
+  on('click', '#modal_backdrop', () => {
```

Comment:
Move this out to the other event handlers, or every time someone brings up the flagging interface you'll be re-binding this, so the event will fire multiple times on close.

Reply:
This is already in in the onPageLoad() section.  So it should only fire the first time someone arrives at the site.  I'm not sure what you want me to do.